src/test/compile-fail/divide-by-zero.rs

```diff
@@ -0,0 +1,8 @@
+// Copied from `test/run-fail`.
+// Mir const-prop and const-folding should detect this error at compile time.
+
+// error-pattern:attempt to divide by zero
+fn main() {
+    let y = 0;
+    let _z = 1 / y;
```

Comment:
and add a `//~ ERROR: attempt to divide by zero` here

Reply:
Does that have an effect outside of ui tests?